packages/parser/tests/lib/services.test.ts

```diff
@@ -32,18 +39,17 @@ function createConfig(filename: string): ParserOptions {
 // Tests
 //------------------------------------------------------------------------------
 
-describe('services', () => {
-  const program = createProgram(path.resolve(FIXTURES_DIR, 'tsconfig.json'));
-  testFiles.forEach(filename => {
-    const code = fs.readFileSync(path.join(FIXTURES_DIR, filename), 'utf8');
-    const config = createConfig(filename);
-    const snapshotName = formatSnapshotName(filename, FIXTURES_DIR, '.ts');
-    it(snapshotName, createSnapshotTestBlock(code, config));
-    it(`${snapshotName} services`, () => {
-      testServices(code, config);
-    });
-    it(`${snapshotName} services with provided program`, () => {
-      testServices(code, { ...config, program });
-    });
+const program = createProgram(path.resolve(FIXTURES_DIR, 'tsconfig.json'));
+
+describe.for(testFiles)('services', async filename => {
+  const code = await fs.readFile(path.join(FIXTURES_DIR, filename), 'utf8');
+  const config = createConfig(filename);
+  const snapshotName = formatSnapshotName(filename, FIXTURES_DIR, '.ts');
```

Comment:
nit: it was already wrong, but these really should be setup in a `beforeAll` hook

if you ever `describe.skip` this for example, it'll still run these 3 lines

Reply:
makes sense to me 👍